icalendar/ICalendar.g4

```diff
@@ -306,16 +306,15 @@ daylightc
 
 // 3.8.1.1 - Attachment
 attach
-    : k_attach attachparam* (
-        COL uri
-        | SCOL k_encoding ASSIGN k_base D6 D4 SCOL k_value ASSIGN k_binary COL binary
-    ) CRLF
-    ;
+   : k_attach attachparam* COL (binary | uri) CRLF
```

Comment:
Please use 4 spaces for indents instead of 3 as it's used for the entire grammar.

Reply:
Done 👍